docker/cyclus-deps/Dockerfile

```diff
@@ -24,16 +24,19 @@ CMD [ "/bin/bash" ]
 # apt packages
 #
 RUN apt-get update && \
-    apt-get install -y gcc vim nano && \
+    apt-get install -y g++ gcc vim nano && \
```

Comment:
is this required because of the switch to ubuntu?

Reply:
Actually it is probably not required... as we install the condo version of gxx and gcc later